modules/core/server/views/layout.server.view.html

```diff
@@ -56,7 +56,7 @@
 
   <!--Embedding The User Object-->
   <script type="text/javascript">
-    var user = {{{ user }}};
+    var user = {{ json user }};
```

Comment:
I think helper is a bit overkill here, how about just doing:

```
var user = {{#if user}}{{{ user }}}{{else}}null{{/if}};
```

(I know, {{}{{}}} — handlebars...)